modules/hello-world/pages/overview.adoc

```diff
@@ -67,11 +67,11 @@ And take a look at the xref:howtos:working-with-collections.adoc[developer previ
 .Practical Howto Docs
 
 [.content]
-Connect to our services -- xref:howtos:kv-operations.adoc[data (KV)]; 
```

Comment:
Can you turn off this sort of auto-blank-space-attack when doing large PRs,
it makes them very noisy.

Reply:
I think it's a default setting in VSCode, can probably turn it off...
I've resorted to "Hide whitespace changes" in github view on occasion btw.